src/bidiTab/mapperTabPage.ts

```diff
@@ -17,8 +17,75 @@
 import {LogType} from '../utils/log.js';
 
 /** HTML source code for the user-facing Mapper tab. */
-const mapperPageSource =
```

Comment:
It was minimised to reduce the Mapper size, as this string will be delivered without any compression. This increases the Mapper size for `518` bytes (`1484` - `966`) .

Reply:
Oh, I see. Reverted.